CIPs/0015.md

```diff
@@ -0,0 +1,59 @@
+# CIP [0015]: Hot standby for validators
+
+- Date: 2020-06-10
+- Author: @githubhandle
```

Comment:
hi @kevjue  thank you for submitting this CIP, I would like you to update it to follow the newly activated CIP process which has it's newly updated template here: https://github.com/celo-org/celo-proposals/blob/master/CIPs/cip-template.md

Also, if the CIP is in Draft status, I'd be able to merge the proposal into the repo.